Makefile

```diff
@@ -10,6 +10,7 @@ export PATH := $(CCTOOLS_INSTALL)/bin/:$(PATH)
 export CCTOOLS_VERSION=7.7.2
 export HYDRA_LAUNCHER=fork
 export OMPI_MCA_rmaps_base_oversubscribe=yes
+export RADICAL_TEMP_SANDBOX=True
```

Comment:
i think this variable is not interpreted as a Python bool so it's maybe misleading to use `True` here in case someone thinks `False` would do something opposite.

Reply:
I think everything to do with RADICAL_TEMP_SANDBOX can be removed from this PR now, because I think the problem it was solving was fixed in a different way by moving away from radical SAGA?